api/app/src/main/kotlin/packit/service/BasePacketService.kt

```diff
@@ -1,25 +1,28 @@
 package packit.service
 
-import org.springframework.http.HttpStatus
+import org.springframework.core.io.InputStreamResource
+import org.springframework.http.*
 import org.springframework.stereotype.Service
 import packit.exceptions.PackitException
 import packit.model.Packet
+import packit.model.PacketMetadata
 import packit.repository.PacketRepository
 import java.security.MessageDigest
 import java.time.Instant
 
 interface PacketService
 {
     fun getPackets(): List<Packet>
-    fun getPacket(id: String): Packet
     fun getChecksum(): String
     fun importPackets()
+    fun getMetadataBy(id: String): PacketMetadata
```

Comment:
The corresponding methods in the `OutpackServer` class are called `getMetadataById` and `getFileByHash` - can we use the same naming convention here for consistency?